frontend/pages/hosts/HostDetailsPage/SoftwareListRow/_styles.scss

```diff
@@ -0,0 +1,43 @@
+.software-list-row {
+  line-height: 38px;
+  border-bottom: 1px solid $ui-borders;
+
+  &__type {
+    max-width: 280px;
+  }
+
+  &__installed-version {
+    max-width: 280px;
+  }
+
+  &:last-child {
+    border-bottom: 0;
+  }
+
+  &--selected {
+    background-color: $core-blue-label-down;
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: $core-blue-label-over;
+  }
+
+  &:active,
+  &:focus {
+    outline: none;
+  }
```

Comment:
I don't think we need these styles because the "Software" table is read-only for now. We also don't need the `.form-field` styles.